sda-commons-starter-web/src/main/resources/org/sdase/commons/spring/boot/web/monitoring/tracing.properties

```diff
@@ -1,3 +1,7 @@
 management.tracing.enabled=true
 management.tracing.propagation.type=b3,w3c
-management.tracing.sampling.probability=1.0
\ No newline at end of file
+management.tracing.sampling.probability=1.0
+management.otlp.tracing.endpoint=http://grafana-agent-traces.monitoring:4317
+management.otlp.tracing.compression=none
+management.otlp.tracing.timeout=10s
+management.tracing.grpc.enabled=true
```

Comment:
Is this what works in our clusters out of the box?

Reply:
Yes, I tested in the services. With these default properties the teams will not have to change any var to enable tracing.